bitnami/harbor/templates/_helpers.tpl

```diff
@@ -530,6 +541,14 @@ Return the proper Docker Image Registry Secret Names
 {{- include "common.images.pullSecrets" (dict "images" (list .Values.core.image .Values.portal.image .Values.jobservice.image .Values.clair.server.image .Values.clair.adapter.image .Values.chartmuseum.image .Values.trivy.image .Values.notary.server.image .Values.notary.signer.image .Values.registry.server.image .Values.registry.controller.image .Values.nginx.image .Values.volumePermissions.image) "global" .Values.global) -}}
 {{- end -}}
 
+{{/*
+Return the metrics port name
+*/}}
+{{- define "harbor.metricsPortName" -}}
+{{- printf "metrics" -}}
+{{- end -}}
```

Comment:
I don't find this helper really useful. It's defining a hardcoded "metrics" values, therefore I'll remove it.